rlw.sh

```diff
@@ -112,8 +112,8 @@ roblox-install () {
 			wget -N -r --no-parent -Aexe http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/ -nd -P /tmp/Firefox-Setup/
 			WINEDLLOVERRIDES="winebrowser.exe,winemenubuilder.exe=" rwine /tmp/Firefox-Setup/*.exe /SD | zenity \
 				--window-icon="$RBXICON" \
-				--title='Installing Mozilla Firefox' \
-				--text='Installing Mozilla Firefox Browser ...' \
+				--title='Installing (Windows) Mozilla Firefox' \
+				--text='This is so you can launch games.\nDo not worry, this will not mess up your current Linux FireFox installation.' \
```

Comment:
This doesn't sound professional. The old language is concise enough, and Firefox is spelt with only the first F capitalized.